tools/rust_analyzer/rust_project.rs

```diff
@@ -89,10 +92,12 @@ pub struct Dependency {
 }
 
 pub fn generate_rust_project(
+    sysroot: &str,
     sysroot_src: &str,
     crates: &BTreeSet<CrateSpec>,
 ) -> anyhow::Result<RustProject> {
     let mut project = RustProject {
+        sysroot: Some(sysroot.into()),
```

Comment:
Is `sysroot` a valid `rust-project.json` attribute?
I don't see it in the documentation:
https://rust-analyzer.github.io/manual.html#non-cargo-based-projects

Reply:
I had the same question but appears to be based on local functional testing! I'll have to file an issue updating the docs to communicate this.

Reply:
https://github.com/rust-lang/rust-analyzer/issues/13608